test/tests/bcs/coupled_dirichlet_bc/coupled_dirichlet_bc.i

```diff
@@ -0,0 +1,110 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  nx = 10
+  ny = 10
+[]
+
+[Variables]
+  [./u]
+    order = FIRST
+    family = LAGRANGE
+  [../]
+
+  [./v]
+    order = FIRST
+    family = LAGRANGE
+  [../]
+[]
+
+[Kernels]
+  [./diff_u]
+    type = Diffusion
+    variable = u
+  [../]
+
+  # Adding a CoupledForce term induces a coupling between the u and v
+  # equations.  Is this enough to trigger the off-diagonal Jacobian
+  # coupling?
+  [./coupled_force_u]
```

Comment:
> On Jan 22, 2015, at 8:46 AM, David Andrs <notifications@github.com> wrote:
> 
> In test/tests/bcs/coupled_dirichlet_bc/coupled_dirichlet_bc.i:
> 
> > +  [./v]
> > +    order = FIRST
> > +    family = LAGRANGE
> > +  [../]
> > +[]
> > +
> > +[Kernels]
> > +  [./diff_u]
> > +    type = Diffusion
> > +    variable = u
> > +  [../]
> > +
> > +  # Adding a CoupledForce term induces a coupling between the u and v
> > +  # equations.  Is this enough to trigger the off-diagonal Jacobian
> > +  # coupling?
> > +  [./coupled_force_u]
> It should not be. CoupleableMooseVariableDependencyIntermediateInterface (which NodalBCs do have) should take care of filling in the appropriate vector(s)...
> 

Yeah this comment is wrong... I will remove it, it was a note to self during debugging.